gui/src/installer/view.rs

```diff
@@ -422,18 +422,18 @@ pub fn import_descriptor<'a>(
     )
 }
 
-pub fn signer_xpubs(xpubs: &[String]) -> Element<Message> {
+pub fn signer_xpubs<'a>(
+    xpubs: &'a [String],
+    words: &'a [&'static str; 12],
+) -> Element<'a, Message> {
     Container::new(
         Column::new()
             .push(
                 Button::new(
                     Row::new().align_items(Alignment::Center).push(
                         Column::new()
-                            .push(text("This computer").bold())
-                            .push(
-                                text("Derive a key from a mnemonic stored on this computer")
-                                    .small(),
-                            )
+                            .push(text("Generate a new mnemonic").bold())
+                            .push(text("Beware to backup the mnemonic as it will be not stored yet on the computer.").small().style(color::GREY_3))
```

Comment:
If the mnemonic will never be stored, you can remove the "yet".

I would also use "back up" as two words.

Also, it might be worth splitting the string across multiple lines in order to ensure `clippy` works as sometimes it skips sections with long strings.